lib/acme_client_registration.rb

```diff
@@ -0,0 +1,25 @@
+require 'acme-client'
+
+class AcmeClientRegistration
+
+  class NoEmailError < StandardError; end;
+
+  DEBUG_ENDPOINT = "https://acme-staging.api.letsencrypt.org/"
+  LIVE_ENDPOINT  = "https://acme-v01.api.letsencrypt.org/"
+
+  attr_reader :endpoint, :client
+
+  def initialize(options = {})
+    raise NoEmailError if ENV['CONTACT_EMAIL'].empty?
+    if options[:debug]
```

Comment:
what's your feeling on the term debug? As mentioned in a different thread I feel like debug should be more verbose output with the live api. Test/Demo etc might be better terms/